### PR TITLE
Add metadata to indexes

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds metadata to the index files, for related content feature.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Related to https://github.com/timescale/web-documentation/pull/175

### CODE REVIEW

**Comment by Loquacity** [comment]
@helrabelo Can you please check this over before I get too far?

File: timescaledb/how-to-guides/page-index/page-index.js [hunk 1]
```diff
@@ -7,14 +7,44 @@ module.exports = [
       {
         title: "Install TimescaleDB",
         href: "install-timescaledb",
+        metaTags: [
```
Comment (helrabelo) [inline timescaledb/how-to-guides/page-index/page-index.js:10]: The result would be this:

Comment (Loquacity) [reply timescaledb/how-to-guides/page-index/page-index.js:10]: ??

**Comment by Loquacity** [comment]
@jacobprall Can you please check this?

**Comment by Loquacity** [comment]
Now I know what I'm doing, the rest should go much faster! 😅

**Comment by Loquacity** [comment]
@ryanbooz @jacobprall Can we merge this one now?

File: timescaledb/how-to-guides/page-index/page-index.js [hunk 2]
```diff
@@ -7,14 +7,24 @@ module.exports = [
...
+            tags: ['tsc', 'install'],
+            keywords: ['install', 'Timescale Cloud'],
+            excerpt: ['Install Timescale Forge']
           },
           {
             title: "Timescale Cloud",
-            href: "installation-cloud"
+            href: "installation-cloud",
+            tags: ['mst', 'install'],
+            keywords: ['install', 'Managed Service for TimescaleDB'],
```
Comment (jacobprall) [inline timescaledb/how-to-guides/page-index/page-index.js:26]: I think the team feels strongly we avoid using this verbiage until late September!

Comment (Loquacity) [reply timescaledb/how-to-guides/page-index/page-index.js:26]: Yep, I'll go through and revert that 👍

**Review by jacobprall — APPROVED** [review]

**Pull request merged**